packages/react-storage/src/components/StorageBrowser/actions/handlers/listLocationItems.ts

```diff
@@ -62,13 +62,28 @@ const parseItems = (
 const parseExcludedPaths = (paths: string[] | undefined): LocationItemData[] =>
   paths?.map((key) => ({ key, id: crypto.randomUUID(), type: 'FOLDER' })) ?? [];
 
+export const filterDotItems = (
```

Comment:
`parseItems` already filters out items with `path` values equivalent to `/` online 45 of this file, we either need to replace that filtering or extend it to include the additional values in this util